### PR TITLE
Feat/additional pack

### PR DESCRIPTION
**Pull request by luketd**
Added Ruby Pack to packs you can buy
- Limited to 1 Pack a day
- Can only be purchased on the shop (no subscription)
- Contains 1 Ruby and 5 other cards at different rarities

Pack Shop Changed
- Added capability of filtering between Base and Ruby Pack

### CODE REVIEW

File: services/packService.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ class PackService {
         'The base trading card pack. Contains 6 cards ranging from Bronze to Hall of Fame rarity.',
       imageUrl: '/images/base-pack-cover.png',
       price: 50000,
-      priceLabel: '50k',
+      priceLabel: '500k',
```
Comment (GuthrieW) [inline services/packService.ts:19]: Why was this changed?

Comment (luketd) [reply services/packService.ts:19]: oh lmao typo

**Review by GuthrieW — APPROVED** [review]
